src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeAugments.cs

```diff
@@ -212,6 +215,10 @@ public static unsafe object LoadValueTypeField(object obj, int fieldOffset, Runt
         public static unsafe object LoadPointerTypeField(object obj, int fieldOffset, RuntimeTypeHandle fieldType)
```

Comment:
There is also `LoadPointerTypeFieldValueFromValueType`. Does it need fixing too?

Reply:
Yeah, looks like it.